src/platform/silabs/EFR32/wifi/wfx_host_events.h

```diff
@@ -141,6 +141,11 @@ typedef struct __attribute__((__packed__)) sl_wfx_mib_req_s
 #define MAX_JOIN_RETRIES_COUNT 5
 #endif
 
+// WLAN retry time intervals in milli seconds
+#define WLAN_MAX_RETRY_TIMER 30000
+#define WLAN_MIN_RETRY_TIMER 1000
+#define WLAN_RETRY_TIMER 5000
```

Comment:
add a `_MS` suffix to these to make unit of measurement clear. Then people do not need to navigate to read comments and there is less likelyhood of mistakes.

Reply:
Thanks for your input, renamed the above macros as per the suggestions.